management-app/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { ROUTES } from '../../constants';
+import './Header.css';
+
+function Header() {
+  return (
+    <header className="App-header">
+      <nav className="nav-links">
+        {ROUTES.map((el) => {
+          if (el.id !== 5) {
+            return (
+              <NavLink key={el.id} to={el.routePath}>
+                {el.title}
+              </NavLink>
+            );
+          }
+          return '';
+        })}
```

Comment:
Something like this:
```suggestion
        {ROUTES
          .filter(({ routePath }) => routePath !== PATHS.NOT_FOUND)
          .map(({ routePath, title }) => (
            <NavLink key={routePath} to={routePath}>
              {title}
            </NavLink>
          ))
        }
```

Reply:
Yes, it looks much better. I'll implement your variant.

Reply:
done commit bede5b3deaa250666fe4176f2b190ea8524e4769